### ISSUE TITLE
Can't match backslash only

### ISSUE THREAD

**Issue by bouncepaw** [Genivia/RE-flex#69]
I want to match a backslash. This does not match:

```c
"\\" printf("backslash");
```

Nor does any of the patterns below:

```c
\\
\x5c
\x5C
"\x5c"
"\x5C"
\Q\\E
\Q\\\E
```

The only pattern that matches is `[\\]`. I think it should not be like that.

I'm using the latest version of reflex (in fact, I cloned it yesterday (2020-03-30)). My options:

```c
%option unicode
%option flex
%option nodefault
%option fast
%option main
```

**Comment by genivia-inc** [comment]
The problem appears to be related to a C++ comment in the generated code, not the regex or the regex matcher, which works fine. The C++ comment contains the regex, where the `\\` causes the comment to continue on the next line:
~~~
echo.l:51:40: warning: multi-line // comment [-Wcomment]
          case 1: // rule at line 20: \\
                                       ^
1 warning generated.
~~~
This may comment out the action and thus prevent it from being executed.

A minor tweak will be made to the generated code to fix this in the upcoming release 1.6.6.

**Comment by bouncepaw** [comment]
What a funny bug :) Thanks for pointing it out!

**Comment by genivia-inc** [comment]
Fixed in 1.6.6.
